packages/common/src/ContractInteractor.ts

```diff
@@ -230,6 +229,13 @@ export class ContractInteractor {
     if (block.baseFeePerGas != null) {
       this.logger.info('Network supports Type 2 Transactions (EIP-1559). Checking RPC node \'eth_feeHistory\' method')
       try {
+        // hard-coded default values as these are not used but compared to 'gasPrice' and logged
+        const { baseFeePerGas, priorityFeePerGas } = await this.getGasFees(5, 50)
```

Comment:
just read the description of the PR, which is the only place that currently give meaning to these "5" and "50".... please make them constants.